graph/src/components/store.rs

```diff
@@ -1047,6 +1047,25 @@ pub trait ChainStore: Send + Sync + 'static {
     ) -> Result<Option<EthereumBlock>, Error>;
 }
 
+pub trait EthereumCallCache: Send + Sync + 'static {
+    /// Cached return value.
+    fn get_call(
```

Comment:
I was going to say that `get` (and `set`) would be enough here, seeing how the trait is called `EthereumCallCache`, but I assume that you called it `get_call` so it doesn't conflict with `Store.get`?

Reply:
That's exactly what happened, I tried just `get` but it started conflicting.